k8-util/helm/fluvio-app/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: fluvio-app
 description: A Helm chart for Fluvio
 type: application
-version: 0.9.0
+version: 0.9.13
```

Comment:
why bump to 0.9.13?  Chart version is template structure version not tied to content.

Reply:
that's what `make` have set up, could be reverted